version.sbt

```diff
@@ -1,2 +1,2 @@
-version       in ThisBuild := "0.19.1-SNAPSHOT"
+version       in ThisBuild := "0.21.0-M1"
```

Comment:
Rho has largely, by change, had the same version number as http4s, but nothing strickly ties us.  Its actually convenient to not be have the same number because we can release breaking changes out of cycle with Http4s.  I think I would rather the next version be 0.20... unless people think the different is to confusing to end users.

Reply:
@zarthross I think it's fine to go out of step with http4s as long as we put the http4s version used in bold, on the first line of each release description, and in other relevant parts of documentation.